schema/any_test.go

```diff
@@ -120,6 +120,14 @@ func TestAny(t *testing.T) {
 			serialized, err := anyType.Serialize(val.unserialized)
 			assert.NoError(t, err)
 			assert.Equals(t, serialized, val.serialized)
+
+			unserialized2, err := anyType.Unserialize(serialized)
+			assert.NoError(t, err)
+			serialized2, err := anyType.Serialize(unserialized2)
+			assert.NoError(t, err)
+			// test unserialize and serialize are reversible
+			assert.Equals(t, unserialized2, unserialized)
+			assert.Equals(t, serialized2, serialized)
```

Comment:
I think that all we have to do here is check that the value of `unserialized2` is equal to the value of `unserialized` (from line 115) -- if that's true, then we don't need to re-serialize the value _again_, because we did that at line 120 and, by this point, we already know that that works.

By the way, I note that we're calling `t.Run()` here inside a loop and referencing `val` from the outer scope...don't we need to "memoize" (or whatever the term is) `val` (i.e., don't we need to make an inner-scope copy of it to prevent it from changing out from under us if we're running in parallel?)?  Or, are we sure that we're not running parallel here?

Reply:
okay i removed `serialized2` creation and equality assertion